flink-cdc-connect/flink-cdc-pipeline-connectors/flink-cdc-pipeline-connector-mysql/src/main/java/org/apache/flink/cdc/connectors/mysql/source/MySqlDataSourceOptions.java

```diff
@@ -281,4 +281,12 @@ public class MySqlDataSourceOptions {
                     .withDescription(
                             "List of readable metadata from SourceRecord to be passed to downstream, split by `,`. "
                                     + "Available readable metadata are: op_ts.");
+
+    @Experimental
+    public static final ConfigOption<Boolean> PARSE_ONLINE_SCHEMA_CHANGES =
+            ConfigOptions.key("scan.parse.online.schema.changes")
```

Comment:
We can add `enabled` to key name to indicate that the configOption is boolean type.